desktop/src/preload/api/transfer/parseXlsxFileList.ts

```diff
@@ -20,7 +20,21 @@ export const parseXlsxFileList = (
               const accessionCell = sheet.B3?.v; // B3 cell value
               const applicationCell = sheet.B4?.v; // B4 cell value
 
-              if (accessionCell && applicationCell) {
+              if (
+                accessionCell &&
+                applicationCell &&
+                accessionCell.toString() !== "" &&
+                applicationCell.toString() !== "" &&
+                !Number.isNaN(
+                  Number(
+                    accessionCell
+                      .toString()
+                      .replace("-", "")
+                      .replaceAll(" ", "a")
```

Comment:
is there a reason accessionCell is treated differently? replacing all spaces to 'a' seem random. What's the expected format of accession number?

Reply:
Added the replaceAll to the application number. The reason for converting spaces to a is because Number.NaN doesnt fail when there is spaces.